cmd/deploy/deploy.go

```diff
@@ -327,6 +326,8 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		return err
 	}
 
+	oktetoLog.SetStage("")
```

Comment:
This is just moved a bit below because `setDeployOptionsValuesFromManifest` function validates some things regarding the manifest, so if it fails, it should belong to `Load manifest` stage